packages/deployed-backend-client/src/deployed_backend_client.ts

```diff
@@ -103,6 +106,16 @@ export class DefaultDeployedBackendClient implements DeployedBackendClient {
     };
   };
 
+  private isSandboxStack = (stackName: string | undefined): boolean => {
+    const mainStackNamePrefix = 'amplify-';
```

Comment:
Open to suggestions on where to export these hard coded strings from in case they can be shared. There was an early attempt here https://github.com/aws-amplify/amplify-backend/pull/258#discussion_r1336334897 that didn't feel right so we ended up removing it.

Reply:
We could use [BackendIdentiferConversions.fromStackName()](https://github.com/aws-amplify/amplify-backend/blob/main/packages/platform-core/src/backend_identifier_conversions.ts#L18) and check for `type === 'sandbox'` in the return value

Reply:
+1 to Ed's suggestion.